glossary.yaml

```diff
@@ -99,7 +99,7 @@ terms:
     name: "Private Network"
   - definition: "A blockchain application is a software application that has integrated a blockchain in its architecture as the storage layer for part of, or, all of its data. This includes software applications that do not own their own contract on the blockchain and instead only interact with the system contracts of the blockchain."
     name: "Blockchain Application"
-  - definition: "A block is considered irreversible (i.e. immutable) on an EOSIO based blockchain when 2/3rd of the currently elected block producers have acknowledged it."
+  - definition: "Irreversible blocks are blocks that contain confirmed, final transactions. A block is considered irreversible (i.e. immutable) on an EOSIO-based blockchain when 2/3rd of the currently elected block producers acknowledge it."
```

Comment:
correction: it is 2/3rds plus one
suggestion: use term "supermajority" in this context
consider:
when 2/3rd of the currently elected block producers acknowledge it. --> when a supermajority consisting of 2/3rds plus one of the currently elected block producers have confirmed the block.

Reply:
Suggestion applied. I liked the word **_supermajority_**.